hyperwallet/utils/apiclient.py

```diff
@@ -112,6 +112,13 @@ def _makeRequest(self,
 
         if response.status_code is 204:
             return {}
+        elif response.status_code is 429:
+            raise HyperwalletAPIException({
+                'errors': [{
+                    'code': 'TOO_MANY_REQUESTS',
+                    'message': 'Too many requests sent in a given amount of time.'
```

Comment:
Please change the message to simply say "Too Many Requests". 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429

Reply:
@wmews-hw I've changed the message